pokemongo_bot/step_walker.py

```diff
@@ -19,17 +19,17 @@ def __init__(self, bot, dest_lat, dest_lng):
             dest_lat,
             dest_lng
         )
-
-        self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
+            
+        self.speed = uniform(self.bot.config.walk_min, self.bot.config.walk_max)
 
         self.destLat = dest_lat
         self.destLng = dest_lng
         self.totalDist = max(1, self.dist)
 
         if self.speed == 0:
-            self.steps = 1
-        else:
-            self.steps = (self.dist + 0.0) / (self.speed + 0.0)
+            self.speed = uniform(3,5)
```

Comment:
Nvm it doesn’t move, only MoveToMapPokemon does.
Maybe I misunderstood what he said. Oh well, he’s the the most qualified person for that one. Better wait him.